File: 2169-count-operations-to-obtain-zero/2169-count-operations-to-obtain-zero.py
impl Solution {
    pub fn count_operations(mut num1: i32, mut num2: i32) -> i32 {
        let mut d = 0;
        while num1 != 0 && num2 != 0
        {
            if num1 < num2 
            {
                let c = num2;
                num2 = num1;
                num1 = c;
            }
            num1 -= num2;
            d += 1;
        }
        d
    }
}

File: 2169-count-operations-to-obtain-zero/2169-count-operations-to-obtain-zero.cpp
class Solution {
public:
    int countOperations(int num1, int num2) {
        int d = 0;
        while(num1 > 0 && num2 > 0)
        {
            if (num1 < num2)
            {
                int c = num1;
                num1 = num2;
                num2 = c;
            }
            num1 -= num2;
            d += 1;
        }
        return d;
    }
};